src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
```

Comment:
Like we discussed in class, we need to be obvious when naming our variables. `obj` seems too generic in this case.